Tests/SonarScanner.MSBuild.Test/AnalysisWarning/WarningsSerializerTest.cs

```diff
@@ -41,7 +41,7 @@ public class WarningsSerializerTest
 
         [TestMethod]
         public void SerializeToFile_EmptyFileDirectory_Throws() =>
-            ((Action)(() => WarningsSerializer.Serialize(Array.Empty<Warning>(), string.Empty))).Should().Throw<ArgumentException>().WithMessage("Empty path name is not legal.");
+            ((Action)(() => WarningsSerializer.Serialize(Array.Empty<Warning>(), string.Empty))).Should().Throw<ArgumentException>().WithMessage("Empty path name is not legal.*");
```

Comment:
I think the message naming depends on the net6 distribution - it might change in the future - maybe just verify that the message contains the argument name?